### ISSUE TITLE
v7 removed in favor of v8

### ISSUE THREAD

**Issue by garyking** [h6y3/todoist-ruby#6]
https://github.com/h6y3/todoist-ruby/blob/562521ca448f5638ea0d6b150b1bebdcf8097620/lib/todoist/config.rb#L3

The v7 endpoint was completely removed in favor of the v8 API, FYI.

**Comment by h6y3** [comment]
Ahh that explains why things seem to completely blow up :)

**Comment by garyking** [comment]
I've just copied this gem to my own project and changed that line, for now. That fixed the issue for me. But I don't use any of the endpoints that have changes made to them, so that might not be enough of a fix for the entire library.

**Comment by h6y3** [comment]
I should be able to make the change and run the unit tests.  I use the library as a series of cron jobs that analyze my tasks, assigns and rebases them according to scripts, postpones, etc. so I started seeing issues the last few days but since my work is a series of cron jobs I didn't have time to figure out what was up.  On the v7 endpoint I was getting HTTP 429 errors which I found strange since that is rate limiting when it should have been just an HTTP 400
